src/libcalamaresui/utils/Paste.cpp

```diff
@@ -78,6 +81,14 @@ sendLogToPastebin( QObject* parent, const QString& ficheHost, quint16 fichePort
     QRegularExpression pasteUrlRegex( "^http[s]?://" + ficheHost );
     QString pasteUrlMsg = QString( pasteUrlFmt ).arg( pasteUrlStr );
 
+    QClipboard* clipboard = QApplication::clipboard();
```

Comment:
this patch segment should be subject to the guarding clause below, which validates the data and URL -
i suggest this change:

```
    if ( nBytesRead >= 8 && pasteUrl.isValid() && pasteUrlRegex.match( pasteUrlStr ).hasMatch() )
    {
        QClipboard* clipboard = QApplication::clipboard();
        clipboard->setText(pasteUrlStr, QClipboard::Clipboard);

        if (clipboard->supportsSelection())
        {
             clipboard->setText(pasteUrlStr, QClipboard::Selection);
        }
    }
    else
    {
        cError() << "No data from paste server";
        return QString();
    }
```